website/js/index.js

```diff
@@ -0,0 +1,21 @@
+$(".imgAdd").click(function(){
+  $(this).closest(".row").find('.imgAdd').before('<div class="imgUp"><div class="imagePreview"></div><label class="btn btn-primary">Upload<input type="file" class="uploadFile img" value="Upload Photo" style="width: 0px;height: 0px;overflow: hidden;"></label></div>');
+});
+$(function() {
+    $(document).on("change",".uploadFile", function()
+    {
+    		var uploadFile = $(this);
+        var files = !!this.files ? this.files : [];
+        if (!files.length || !window.FileReader) return; // no file selected, or no FileReader support
+ 
+        if (/^image/.test( files[0].type)){ // only image file
+            var reader = new FileReader(); // instance of the FileReader
+            reader.readAsDataURL(files[0]); // read the local file
+ 
+            reader.onloadend = function(){ // set image data as background of div
+                //alert(uploadFile.closest(".upimage").find('.imagePreview').length);
+uploadFile.closest(".imgUp").find('.imagePreview').css("background-image", "url("+this.result+")");
```

Comment:
It'd be more appropriate to have an `img` element with an empty `src` attribute that you dynamically update via JS here, rather than mutating the CSS property.

* It's faster
* Semantically correct: the image should be an image, which will make it easier for screen readers to assess
* You can auto-set the width of the image to an appropriate value for the screen-size - e.g. fix it at `width="400px"` to constrain the top edge. Uploaded images from a phone are likely to be > 3000px wide - and ~4500px tall (= ~13 megapixels).